packages/lit-dev-content/site/css/docs.css

```diff
@@ -530,6 +534,13 @@ table.directory tr.subheading {
   background: #d6d6d6;
 }
 
+/* Labs icon */
+#docsNav img.labs {
+  width: 1.25em;
```

Comment:
it's not in line with the rest of the style, but please use `rem`

Reply:
I kept it as `em` so it would scale with the font-size. Do we not want that?

Reply:
I don't think we actually use it and it will likely only cause confusion down the line